mariadb_kernel/kernel.py

```diff
@@ -121,13 +121,26 @@ def do_execute(
                 continue
 
             self._update_data(result)
-
-            display_content = {
-                "data": {"text/html": self._styled_result(result)},
-                "metadata": {},
-            }
+            result_str = str(result)
             if not silent:
-                self.send_response(self.iopub_socket, "display_data", display_content)
+                if (
+                    not result_str
+                ):  # empty result set, should, show success info to user
+                    display_content = {
+                        "data": {"text/plain": "Query OK"},
+                        "metadata": {},
+                    }
+                    self.send_response(
+                        self.iopub_socket, "display_data", display_content
+                    )
+                else:
+                    display_content = {
+                        "data": {"text/html": result_str},
```

Comment:
Here, you missed the _styled_result call (see above in deleted lines).

Reply:
I got it. 😐

Reply:
Actually the _styled_result call needs to be here, i.e. `"..html": self._styled_result(result_str)`, and the `display_content` dict below needs to be left alone because _styled_result acts on a HTML string directly, not a dictionary type.
It is my fault as it is hard for you to figure out what's the use case for this, we don't have a test for the bug I fixed with this styled_result function. I will add a test this evening. Thanks for the patience.

And because we've both just seen how important a test case is, can you please also add a test case for this PR in `test_mariadbclient.py`? The test should run a `use test;` statement for instance, and the expected result should be `Query OK`. I promise this is the last request before merge, thanks a lot for the patience 🙇

Reply:
Maybe fix it and I add a test for empty-result queries. Unit test is important🤣. This is my first time open a pull request. Maybe I ignore something or make mistake. Thanks for a lot of patience with me😀.